java-library/src/main/java/com/microsoft/azure/functions/dapr/annotation/DaprBindingOutput.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.functions.dapr.annotation;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import com.microsoft.azure.functions.annotation.CustomBinding;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.ElementType;
+
+/**
+ * Attribute to specify parameters for the Dapr output bindings.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.PARAMETER)
+@CustomBinding(direction = "out", name = "daprBindingOutputMessage", type = "daprBinding")
+public @interface DaprBindingOutput {
+    /**
+     * Dapr runtime endpoint.
+     */
+    String daprAddress() default "";
+
+    /**
+     * Configured name of the binding.
```

Comment:
Since binding word is overused, maybe we can simplify it.
```suggestion
Name of the Dapr binding, should be same as the name specified in the Dapr component YAML.
```

We can also share the same doc comment (should be same as ...) in other interface docs.

Reply:
We can also rename `bindingName` to `daprBindingName` in both output binding and trigger.

Reply:
It is consistent with Dapr Extension naming, and changing it here is not possible until we change it in extension and all other languages.